UEBPrep/app/views/cards/_footer.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= social_share_button_tag("A CARD") %>
```

Comment:
Does this work with any of the social media accounts, or laying the groundwork for the future?